app/javascript/CommentSubscription/CommentSubscription.jsx

```diff
@@ -38,39 +38,6 @@ export class CommentSubscription extends Component {
     this.state = initialState;
   }
 
-  componentDidUpdate() {
-    const { showOptions } = this.state;
-
-    if (showOptions) {
-      window.addEventListener('scroll', this.dropdownPlacementHandler);
-      this.dropdownPlacementHandler();
-    } else {
-      window.removeEventListener('scroll', this.dropdownPlacementHandler);
-    }
-  }
-
-  componentWillUnmount() {
-    window.removeEventListener('scroll', this.dropdownPlacementHandler);
-  }
-
-  dropdownPlacementHandler = () => {
-    const { base: element } = this.dropdownElement;
-
-    // Reset the top before doing any calculations
-    element.style.bottom = '';
-
-    const { bottom: dropDownBottom } = element.getBoundingClientRect();
-    const { height } = this.buttonGroupElement.base.getBoundingClientRect();
-
-    if (
-      Math.sign(dropDownBottom) === -1 ||
-      dropDownBottom > window.innerHeight
-    ) {
-      // The 4 pixels is the box shadow from the drop down.
-      element.style.bottom = `${height + 4}px`;
-    }
-  };
-
```

Comment:
So one of our dropdowns (for Discussion's _Subscribe_ button) was actually doing this repositioning but it was completely custom for some reason (I guess it was created BEFORE the `reverse` magic). So I removed it and updated the component itself with `reverse` magic handler.